test/e2e/webdriver/firefox.js

```diff
@@ -62,31 +58,13 @@ class FirefoxDriver {
     this._driver = driver
   }
 
-  /**
-   * Initializes the driver
-   * @returns {Promise<void>}
-   */
-  async init() {
-    await this._driver
-      .getExecutor()
-      .defineCommand(
-        GeckoDriverCommand.INSTALL_ADDON,
-        'POST',
-        '/session/:sessionId/moz/addon/install',
-      )
-  }
-
   /**
    * Installs the extension at the given path
    * @param {string} addonPath - the path to the unpacked extension or XPI
    * @returns {Promise<string>} the extension ID
    */
   async installExtension(addonPath) {
-    const cmd = new Command(GeckoDriverCommand.INSTALL_ADDON)
```

Comment:
We had to install the addon manually before because `selenium-webdriver` didn't support installing Firefox addons from a _folder_ - it only supported an `xpi` file.

Now that it supports `.zip` as well, we can get rid of this initialization and manual install.